homework_12/src/components/list/List.js

```diff
@@ -1,32 +1,20 @@
 import './list.scss';
 
 const List = (props) => {
-  const { from } = props;
-  const { to } = props;
-  const { even } = props;
-  const { odd } = props;
+  const { from, to, even, odd } = props;
   const array = [];
 
   for (let i = +from; i <= +to; i++) {
     array.push(i);
   }
-  if (even) {
-    return (
-      <ul>
-        {array.filter(item => item % 2 === 0).map(item => <li key={item}>{item}</li>)}
-      </ul>
-    );
-  }
-  if (odd) {
-    return (
-      <ul>
-        {array.filter(item => item % 2).map(item => <li key={item}>{item}</li>)}
-      </ul>
-    );
-  }
   return (
-    <ul>
-      {array.map(item => <li key={item}>{item}</li>)}
+    <ul>{
+      even
+        ? array.filter(item => item % 2 === 0).map(item => <li key={item}>{item}</li>)
+        : odd
+          ? array.filter(item => item % 2).map(item => <li key={item}>{item}</li>)
+          : array.map(item => <li key={item}>{item}</li>)
+    }
```

Comment:
1) bad practice to use nested `? ... : ... `, do not use it in future without good reason
2) `item => <li key={item}>{item}</li>` why you repeat this function 3 times? Just move it to variable
```
const createLi = item => <li key={item}>{item}</li>
...
...map(createLi);
```